dashboard/app/views/projects/index.html.haml

```diff
@@ -3,10 +3,12 @@
 - projects_data = {}
 - if current_user
   - projects_data[:canViewAdvancedTools] = !(current_user.under_13? && current_user.terms_version.nil?)
-  // Due to inappropriate content, we're only showing non-featured Applab and Gamelab
-  // projects internally to project_validators via the view more link.
-  // TODO Erin B - remove when we have enough featured Applab and Gamelab projects and/or profanity filter in place.
-  - projects_data[:projectValidator] = current_user.permission?  UserPermission::PROJECT_VALIDATOR
+  // Due to risk of inappropriate content, we can hide non-featured Applab
+  // and Gamelab projects via DCDO. Internally, project_validators should
+  // always have access to all Applab and Gamelab projects, even if there is a // limited gallery for others.
+  - limited_project_gallery = DCDO.get('image_moderation', {})['limited_project_gallery']
```

Comment:
Can we reuse or remove the helper we already wrote for this?

https://github.com/code-dot-org/code-dot-org/blob/c9ff8231460b25182f720257dc2bdae7d68c5950/lib/cdo/azure_content_moderator.rb#L172-L176

We may need to make it a static method.  If you decide to remove that one and do this inline, let's make sure we capture the "true by default" behavior.

It might also be nice to make `limited_gallery?` a helper in projects_controller.rb and call that here instead of bothering with DCDO and UserPermission right here in the view.